Quality-Assistance.md

```diff
@@ -0,0 +1,96 @@
+## Quality Assistance
```

Comment:
Here and in [Confluence](https://octopushq.atlassian.net/wiki/spaces/IN/pages/892370955/Quality+Assistance+Team), there's a mixture of "Quality Assistance engineer", "Quality Assistance", "quality assurance", and "quality coaching". Do they all mean different things, or are there some synonyms we can eliminate?

Reply:
Good question. As I've talked to people in other scenarios, they talk about the "practice of quality assistance" as a model and "quality coach" as a role.

We don't have to follow that convention. Perhaps we remove everything except "quality coach" and derivatives? What do you think @Waldo000000

Reply:
I think so too. `Quality coaches` provide `quality assistance` in general. 
➕  on